src/tracer.h

```diff
@@ -122,7 +123,16 @@ class TraceManager {
       const std::string& filepath, const InferenceTraceMode mode,
       const TraceConfigMap& config_map);
 
+#if !defined(_WIN32) && defined(TRITON_ENABLE_TRACING)
+  ~TraceManager()
+  {
+    if (global_setting_->mode_ == TRACE_MODE_OPENTELEMETRY) {
```

Comment:
Can this conditional move into CleanupTracer as well?

Reply:
done